joomla/joomla-cms
# [4.0] Associations fixes and Menu improvements

### Summary of Changes

This PR does the following:

- Fix the styling for some menu views
- Removes unused inline CSS from `com_menus`
- Moves rest of inline CSS and JS from `com_menus` to separate files
- Fix JS error in `associations-edit.js` when creating a new menu item


### Testing Instructions

As this is primarily a Javascript focused PR, create a new menu item, set the type and ensure there are no errors.

The CSS stuff is simply a code review
